tox.ini

```diff
@@ -36,7 +36,7 @@ commands =
 
 [flake8]
 exclude =
-    .venv,.git,.tox,*lib/python*,*egg,build
+    .venv,.git,.tox,*lib/python*,*egg,build,deployenv
```

Comment:
I'm not seeing any fails due to 'deployenv'. Doesn't '*lib/python*' exclude any of the source files from the virtual environment?